generator/types_generator.go

```diff
@@ -314,10 +314,23 @@ func (gen *goTypesGenerator) makeParameter(parameter *spec.Parameter) (*types.Ty
 			return enumType, nil
 		}
 
+	} else if parameter.Type == "file" {
+
+		return types.NewFile(strings.Title(parameter.Name), parameter.Required), nil
+
 	} else {
 		typ, err := types.FromSimpleSchema("", &parameter.SimpleSchema, parameter.Required, &parameter.CommonValidations)
```

Comment:
suggestion: the empty string could be replaced by a const with a proper name. It will be simpler to understand what's the meaning of this first parameter is.

Reply:
Thanks for the suggestion. I created a new issue (#8) for that to separated the concerns of this pull request from the ideas of the suggestion.